library/pkcs11_client.c

```diff
@@ -138,6 +138,39 @@ static size_t pkcs11_pk_signature_size( const void *ctx_arg )
     }
 }
 
+static int pkcs11_sign_core( mbedtls_pk_pkcs11_context_t *ctx,
```

Comment:
We want our code to compile under `-Wunused-functions`, so this function needs to be protected by `#ifdef` guards. At this stage the guard is `defined(MBEDTLS_RSA_C)` but when this PR is merged this function will also be needed with `MBEDTLS_ECDSA_C`.

Reply:
Done.